src/components/layout/Banner.jsx

```diff
@@ -1,12 +1,25 @@
-import React from 'react'
-import { AddMovie } from './AddMovie'
-import { SearchMovie } from './SearchMovie'
+import React, { useContext } from 'react';
+import { AddMovie } from './AddMovie';
+import { SearchMovie } from './SearchMovie';
+import { MovieContext } from '../../App';
+import { MovieDetail } from '../MovieDetail';
+
+export const Banner = ({ handleDetail }) => {
+  const movieDetail = useContext(MovieContext);
```

Comment:
As I mentioned in Teams, here you can write a custom hook that will be returning `movideDetail` from the context. Practically speaking it's pointless but will give you some idea on how custom hooks work.

Reply:
Done